src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeSectionsInShellReferenceContributor.kt

```diff
@@ -79,18 +84,20 @@ class SnakemakeSectionsInShellReferenceContributor : PsiReferenceContributor() {
     }
 
     private fun addSectionReferences(
-            element: PsiElement,
+            element: PyStringLiteralExpression,
             sectionMatcher: Matcher,
             sectionReferences: MutableList<PsiReference>
     ) {
         while (sectionMatcher.find()) {
             val sectionName = sectionMatcher.group(1)
             val argumentName = sectionMatcher.group(3)
-            sectionReferences.add(SmkSectionReference(
-                    element as PyStringLiteralExpression,
-                    TextRange(sectionMatcher.start(1), sectionMatcher.end(1)),
-                    if (sectionName.isEmpty()) null else sectionName
-            ))
+            if (sectionName.isEmpty() || ALLOWED_IN_SHELL_WITHOUT_KEYWORDS.any { it.startsWith(sectionName) }) {
```

Comment:
Shows false unresolved error for `wildcards`

![image](https://user-images.githubusercontent.com/72933/62640366-1b678b80-b94a-11e9-9e8e-ba02c4e77cf5.png)